pokemonred_puffer/cleanrl_puffer.py

```diff
@@ -727,6 +727,7 @@ def __enter__(self):
         return self
 
     def __exit__(self, *args):
-        print("Done training. Saving data...")
+        print("Done training. Saving checkpoint...")
+        self.save_checkpoint()
```

Comment:
This was removed on purpose and I'd rather it be under a flag. Currently, we are not running evaluations and checkpoint saving has caused more headaches than solutions. It frequently errors if the wrong flags are set creating confusion for the person running this script.

Reply:
Added save_checkpoint flag to the config, and save_checkpoint() acts accordingly. Fixed the statement as well.